### PR TITLE
D2IQ-64509 Refresh Kommander RBAC (Access Control) docs & screenshots

### PR DESCRIPTION
**Pull request by brandonc**
## Jira Ticket

https://jira.d2iq.com/browse/D2IQ-64509

## Description of changes being made

The UX has changed quite a lot from the original docs. I punched up the operations section a bit and refreshed the tutorial screenshots

## Checklist
- [x] Make sure you change all affected versions (e.g. 1.10, 1.11, 1.12, 1.13).

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing) for more information.

### CODE REVIEW

File: pages/ksphere/kommander/latest/operations/access-control/index.md [hunk 1]
```diff
@@ -6,36 +6,30 @@ menuWeight: 1
...
 
-Role-based authorization can be defined centrally within Kommander to control access to resources on all clusters. The resources are similar to Kubernetes RBAC but with some crucial differences.
+Role-based authorization can be defined centrally within Kommander to control access to resources on all clusters or a subset of clusters. The resources are may seem familiar because they are similar to Kubernetes RBAC but with some crucial differences that make it possible to define the roles and policies once and have them federated to clusters within a given scope:
 
-## Types of Access Control Objects
-
-**Groups** map to groups and user claims from your identity providers.
-
-![Groups](/ksphere/kommander/img/Access-control-groups-table.png)
+- Global or Workspace-level roles and policies federate those resources to a cluster-scoped roles and policies. In the case of workspace-level resources, they only apply to workspace clusters.
```
Comment (natmegs) [inline pages/ksphere/kommander/latest/operations/access-control/index.md]:
```suggestion
- Global or Workspace-level roles and policies federate those resources to cluster-scoped roles and policies. In the case of workspace-level resources, they only apply to workspace clusters.
```

Comment (natmegs) [inline pages/ksphere/kommander/latest/operations/access-control/index.md]: I wonder if it's worth making it extra clear that the global roles/policies will affect all clusters in all workspaces?

File: pages/ksphere/kommander/latest/operations/access-control/index.md [hunk 2]
```diff
@@ -6,36 +6,30 @@ menuWeight: 1
...
 
-Role-based authorization can be defined centrally within Kommander to control access to resources on all clusters. The resources are similar to Kubernetes RBAC but with some crucial differences.
+Role-based authorization can be defined centrally within Kommander to control access to resources on all clusters or a subset of clusters. The resources may seem familiar because they are similar to Kubernetes RBAC but with some crucial differences that make it possible to define the roles and policies once and have them federated to clusters within a given scope:
 
-## Types of Access Control Objects
-
-**Groups** map to groups and user claims from your identity providers.
-
-![Groups](/ksphere/kommander/img/Access-control-groups-table.png)
+- Global or Workspace-level roles and policies federate those resources to cluster-scoped roles and policies. In the case of workspace-level resources, they only apply to workspace clusters.
```
Comment (GeorgiSTodorov) [inline pages/ksphere/kommander/latest/operations/access-control/index.md]: Once we have Workspace-level capitalized and once not capitalized. Shouldn't it be consistent?

File: pages/ksphere/kommander/latest/operations/access-control/index.md [hunk 3]
```diff
@@ -6,36 +6,30 @@ menuWeight: 1
...
+Cluster roles and project roles are named collections of rules defining which verbs can be applied to which resources.
 
-![Project Roles](/ksphere/kommander/img/Project-roles-table.png)
+![Cluster Roles](/ksphere/kommander/img/access-control-cluster-roles.png)
 
-Project Roles
+### Policies
 
-![Project Policies](/ksphere/kommander/img/Project-policies-table.png)
+Cluster policies and project policies bind a Kommander group to any number of roles. All groups that have been defined in the groups tab will be present at the Global, workspace, or project level and ready for you to assign roles to them.
```
Comment (GeorgiSTodorov) [inline pages/ksphere/kommander/latest/operations/access-control/index.md]: Why is only "Global" written with a capital letter? Shouldn't workspace and project be also capital?

File: pages/ksphere/kommander/latest/tutorials/authorize-all-users/index.md [hunk 4]
```diff
@@ -35,7 +35,7 @@ For this group to have an effect we need to connect it to a role, so let's first
 
 ![Adding a read role](/ksphere/kommander/img/tutorial-auth-developer-role.png)
 
-Now that we have everything we can create our policy.
+Now that we have everything we can assign the Read Everything role to the developers group
```
Comment (GeorgiSTodorov) [inline pages/ksphere/kommander/latest/tutorials/authorize-all-users/index.md]: Shouldn't we have dot at the end of the sentence? Also, shouldn't the role name be in quotation marks?

File: pages/ksphere/kommander/latest/operations/access-control/index.md [hunk 5]
```diff
@@ -6,36 +6,30 @@ menuWeight: 1
...
-![Roles](/ksphere/kommander/img/Access-control-roles-table.png)
-
-Roles
+## Types of Access Control Objects
 
-**Policies** bind a group to a role
+### Groups
 
-![Policies](/ksphere/kommander/img/Access-control-policies-table.png)
+You can map group and user claims made by your configured identity providers to Kommander groups by clicking the Groups tab under Global Administration / Identity providers.
```
Comment (GeorgiSTodorov) [inline pages/ksphere/kommander/latest/operations/access-control/index.md]: I think "You can map group and user claims made by your configured identity providers to Kommander groups by clicking the Groups tab under **Administration / Identity providers on a Global-level**." is clearer.

File: pages/ksphere/kommander/latest/operations/access-control/index.md [hunk 6]
```diff
@@ -6,36 +6,30 @@ menuWeight: 1
...
 
-## Types of Access Control Objects
-
-**Groups** map to groups and user claims from your identity providers.
-
-![Groups](/ksphere/kommander/img/Access-control-groups-table.png)
+- Global or Workspace-level roles and policies federate those resources to cluster-scoped roles and policies. In the case of workspace-level resources, they only apply to workspace clusters.
 
-Groups
+- Project-level roles and policies federate namespace-scoped role and policies to project clusters.
```
Comment (GeorgiSTodorov) [inline pages/ksphere/kommander/latest/operations/access-control/index.md]: Shouldn't "role" be "roles".

**Comment by GeorgiSTodorov** [comment]
Sorry if my questions are stupid, English isn't my first language.

File: pages/ksphere/kommander/latest/operations/access-control/index.md [hunk 7]
```diff
@@ -6,36 +6,30 @@ menuWeight: 1
 excerpt: Centrally manage access across clusters
 ---
 
-Role-based authorization can be defined centrally within Kommander to control access to resources on all clusters. The resources are similar to Kubernetes RBAC but with some crucial differences.
+Role-based authorization can be defined centrally within Kommander to control access to resources on all clusters or a subset of clusters. The resources may seem familiar because they are similar to Kubernetes RBAC but with some crucial differences that make it possible to define the roles and policies once and have them federated to clusters within a given scope:
```
Comment (russurquhart1) [inline pages/ksphere/kommander/latest/operations/access-control/index.md]: Role-based authorization can be defined centrally within Kommander to control resource access on all clusters or a subset of clusters. The resources are similar to Kubernetes RBAC but with crucial differences. These resouces  make it possible to define the roles and policies once and have them federated to clusters within a given scope:

File: pages/ksphere/kommander/latest/operations/access-control/index.md (hunk 2 above)
Comment (russurquhart1) [reply pages/ksphere/kommander/latest/operations/access-control/index.md]: I agree, i think it should probably be workspace-level. (But who ever is the expert, please let me know!)

**Pull request merged**